docs/install.rst

```diff
@@ -141,6 +141,8 @@ Requirements
 
 - `PyYAML <https://pyyaml.org>`_ |minimum_pyyaml_version| or later
 
+- `packaging <https://packaging.pypa.io/>`_ |minimum_packaging_version| or later
```

Comment:
This is breaking the documentation build because `|minimum_packaging_version|` isn't defined anywhere.